content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than `2.155`.
+
+Since Jenkins `2.155`, it is possible to run Jenkins with Java 11.
+However, it was greatly simplified since version `2.163`. 
+For the sake of simplicity, we will describe next how to run Jenkins on Java 11 with the most recent version of Jenkins.
+
+NOTE: We do encourage you to only use LTS versions for production purpose. 
+At the time this documentation is writen, there is no LTS version with Java 11 Support.
```

Comment:
Instead of "At the time this documentation is writen" I would use "At the time of this writing"

Reply:
```suggestion
At the time of this writing, there is no LTS version with Java 11 Support.
```